Renaming all occurences of 'ls_registration_daemon' to 'lsregistrationdaemon'.
--- a/bin/ls_registration_daemon.pl
+++ b/bin/ls_registration_daemon.pl
@@ -5,7 +5,7 @@
 
 =head1 NAME
 
-ls_registration_daemon.pl - Registers services (e.g. daemons such as owamp,
+lsregistrationdaemon.pl - Registers services (e.g. daemons such as owamp,
 bwctl) into the global information service.
 
 =head1 DESCRIPTION
@@ -31,7 +31,7 @@
 use Log::Log4perl qw/:easy/;
 
 # set the process name
-$0 = "ls_registration_daemon.pl";
+$0 = "lsregistrationdaemon.pl";
 
 my @child_pids = ();
 
@@ -72,7 +72,7 @@
 }
 
 if ( not $PIDFILE ) {
-    $PIDFILE = "/var/run/ls_registration_daemon.pid";
+    $PIDFILE = "/var/run/lsregistrationdaemon.pid";
 }
 
 ( $status, $res ) = lockPIDFile( $PIDFILE );
@@ -181,7 +181,7 @@
 #initialize the key database
 unless ( $conf{"ls_key_db"} ) {
     $logger->info( "No LS key database found" );
-    $conf{"ls_key_db"} = '/var/lib/perfsonar/ls_registration_daemon/lsKey.db';
+    $conf{"ls_key_db"} = '/var/lib/perfsonar/lsregistrationdaemon/lsKey.db';
 }
 my $ls_key_dbh = DBI->connect('dbi:SQLite:dbname=' . $conf{"ls_key_db"}, '', '');
 my $ls_key_create  = $ls_key_dbh->prepare('CREATE TABLE IF NOT EXISTS lsKeys (uri VARCHAR(255) PRIMARY KEY, expires BIGINT NOT NULL, checksum VARCHAR(255) NOT NULL, duplicateChecksum VARCHAR(255) NOT NULL)');
--- a/doc/README
+++ b/doc/README
@@ -20,7 +20,7 @@
 
 There are several files that may be adjusted manually to configure this service:
 
-  - etc/ls_registration_daemon.conf: 
+  - etc/lsregistrationdaemon.conf: 
   
     Specifies which services to check, how frequently to check them and where
     and how to register their presence
@@ -57,10 +57,10 @@
                                  the daemon will read that file to figure out which address and
                                  port to use.
 
-    The default ls_registration_daemon.conf comes with examples describing how
+    The default lsregistrationdaemon.conf comes with examples describing how
     one might describe various deployment scenarios.
 
-  - etc/ls_registration_daemon-logger.conf:
+  - etc/lsregistrationdaemon-logger.conf:
 
     Contains information related to logging, there are three modes:
     
--- a/etc/ls_registration_daemon-logger.conf
+++ b/etc/ls_registration_daemon-logger.conf
@@ -9,7 +9,7 @@
 
 # Uncomment the following lines to log messages to the specified file.
 log4perl.appender.A1=Log::Dispatch::FileRotate
-log4perl.appender.A1.filename=/var/log/perfsonar/ls_registration_daemon.log
+log4perl.appender.A1.filename=/var/log/perfsonar/lsregistrationdaemon.log
 log4perl.appender.A1.max=7
 log4perl.appender.A1.DatePattern=yyyy-MM-dd
 log4perl.appender.A1.permissions=sub{ 0644; }
@@ -20,11 +20,11 @@
 log4perl.appender.A1.layout.ConversionPattern=%d (%P) %p> %F{1}:%L %M - %m%n
 
 # Uncomment the following lines to enable NetLogger instrumentation to the file
-# "/var/log/perfsonar/ls_registration_daemon-netlogger.log"
+# "/var/log/perfsonar/lsregistrationdaemon-netlogger.log"
 #
 #log4perl.category.NetLogger=DEBUG, A2
 #log4perl.appender.A2=Log::Dispatch::File
-#log4perl.appender.A2.filename=/var/log/perfsonar/ls_registration_daemon-netlogger.log
+#log4perl.appender.A2.filename=/var/log/perfsonar/lsregistrationdaemon-netlogger.log
 #log4perl.appender.A2.mode=append
 #log4perl.appender.A2.layout=Log::Log4perl::Layout::PatternLayout
 #log4perl.appender.A2.layout.ConversionPattern=%m%n
